.scalafmt.conf

```diff
@@ -0,0 +1,47 @@
+version = 3.0.5
+style = defaultWithAlign
+maxColumn = 150
+lineEndings = unix
+importSelectors = singleLine
+runner.dialect = scala213
```

Comment:
Not sure if this has been discussed prior but don#t we want a scala 3 dialect?

Also wondering where this config has come from?